src/functions.c

```diff
@@ -33,6 +33,20 @@
 #include "adlist.h"
 #include "atomicvar.h"
 
+/* Functions flags */
+#define FUNCTION_FLAG_NO_WRITES (1ULL<<0)
+#define FUNCTION_FLAG_ALLOW_OOM (1ULL<<1)
+#define FUNCTION_FLAG_ALLOW_STALE (1ULL<<3)
+#define FUNCTION_FLAG_NO_CLUSTER (1ULL<<4)
```

Comment:
considering EVAL, these should be in the header.

Reply:
I guess to also change the name :+1:

Reply:
Fixed